index.html

```diff
@@ -158,34 +155,34 @@ <h1>About</h1>
                 <div class="clearfix-big"></div>
                 <div class="languages">
                     <h3>Languages</h3>
-                    <p>
-                        • JavaScript (ES5/ES6) <br>
-                        • Ruby <br>
-                        • HTML <br>
-                        • CSS 
-                    </p>
+                    <ul class="flex flex-dir-col ul-format">
+                        <li>JavaScript (ES5/ES6)</li>
+                        <li>Ruby</li>
+                        <li>HTML</li>
+                        <li>CSS</li> 
+                    </ul>
                 </div>
                 <div class="clearfix"></div>
                 <div class="frameworks">
                     <h3>Frameworks</h3>
-                    <p>
-                        • Bootstrap <br>
-                        • Ruby  on rails <br>
-                        • RSpec <br>
-                        • Capibara <br>
-                        • Selenium
-                    </p>
+                    <ul class="flex flex-dir-col ul-format">
+                        <li>Bootstrap</li>
+                        <li>Ruby  on rails</li>
+                        <li>RSpec</li>
+                        <li>Capibara</li>
+                        <li>Selenium</li>
+                    </ul>
                 </div>
                 <div class="clearfix"></div>
                 <div class="skills">
                     <h3>Skills</h3>
-                    <p>
-                        • Codekit <br>
-                        • Codepen <br>
-                        • GitHub <br>
-                        • GitLAb <br>
-                        • Terminal <br>
-                    </p>
+                    <ul class="flex flex-dir-col ul-format">
+                        <li>Codekit</li>
+                        <li>Codepen</li>
+                        <li>GitHub</li>
+                        <li>GitLAb</li>
+                        <li>Terminal</li>
+                    </ul>
                 </div>
```

Comment:
- [ ] Please fix the design of these `ul` tags to match the Figma design. (add bullet points which got removed from `list-style: none`)

| Your design | Figma design |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/62290545/158994605-cce40f30-66c2-4b36-ab52-63dd8a5b5d28.png) | ![image](https://user-images.githubusercontent.com/62290545/158994684-0116d9dc-389f-40b7-b688-2ce8ca92ecd1.png) |